environments/skeleton/{{cookiecutter.environment}}/terraform/variables.tf

```diff
@@ -131,3 +131,7 @@ variable "root_volume_size" {
     type = number
     default = 40
 }
+
+variable "k3s_token" {
```

Comment:
This does need a description here (doesn't in the compute/variables.tf), including that it is set from the ansible auomatically